### ISSUE TITLE
Grey dot on space icon looks like a visual bug

### ISSUE THREAD

**Issue by ara4n** [vector-im/element-web#16994]
Plus, what does it mean?

<img width="65" alt="Screenshot 2021-04-19 at 09 42 05" src="https://user-images.githubusercontent.com/1294269/115207491-90193400-a0f3-11eb-8ed8-d3b19973376d.png">

**Comment by t3chguy** [comment]
that one of your rooms has a grey dot on it

e.g
![image](https://user-images.githubusercontent.com/2403652/115207630-af17c600-a0f3-11eb-9618-48ee3de75032.png)

**Comment by t3chguy** [comment]
Related https://github.com/vector-im/element-web/issues/16836

**Comment by niquewoodhouse** [comment]
Using a 3px border of groupFilterPanel-bg-color (or whatever the space panel uses for bg.  Would only work on notifications in space panel, obviously.  

![image](https://user-images.githubusercontent.com/1587166/117140354-3aa47e80-ada5-11eb-99e1-58acfc9f2961.png)

![image](https://user-images.githubusercontent.com/1587166/117140557-717a9480-ada5-11eb-8e51-98624d485ce3.png)

@t3chguy not sure if we need to do the same/similar to other notifications for sake of simplicity.  Using a 2px border instead here because the other notification dots/badges are much bigger already.

![image](https://user-images.githubusercontent.com/1587166/117141917-08941c00-ada7-11eb-929f-d42bcf196320.png)
![image](https://user-images.githubusercontent.com/1587166/117141953-1184ed80-ada7-11eb-86ce-c679ff03860e.png)


![image](https://user-images.githubusercontent.com/1587166/117141994-1c3f8280-ada7-11eb-9092-f087fc537616.png)
![image](https://user-images.githubusercontent.com/1587166/117142077-35e0ca00-ada7-11eb-8752-8155b30c3bc6.png)

**Comment by t3chguy** [comment]
We first need product to decide if showing grey dots (unread rooms you likely don't care about, e.g name changes) on space panel are something we even want.

Just like we don't expose the grey dot on the app icon, as its too noisy; 
![image](https://user-images.githubusercontent.com/2403652/117143027-49406500-ada8-11eb-8f9f-6505bc648d96.png)

**Comment by nadonomy** [comment]
@t3chguy @niquewoodhouse I've just been tracing through some pros & cons of the grey dots, the headlines I'm arriving at are:

Without the dot
- Pro: Reduces noise for users in high traffic rooms/spaces
- Con: Don't know when there's activity in Spaces, forcing a behaviour of needing to manually hunt

With the dot
- Pro: Know when there's activity in rooms or spaces
- Pro: Leverages affordance from other platforms
- Con: Visual noise for users in high traffic rooms/spaces

My sense is the 'high traffic' users will develop blindness to the dots anyway, leaving the utility for everyone else— so I think we should go ahead with the dot to learn more in the beta.

> (unread rooms you likely don't care about, e.g name changes)

@t3chguy on this— iirc metadata related events (rather than messages) like name changes don't inform the room list decoration, am I misremembering? My assumption was that whatever logic we're using there would be mirrored here.

**Comment by t3chguy** [comment]
> Don't know when there's activity in Spaces, forcing a behaviour of needing to manually hunt

Would a user care that `Bob` changed his name to `Bob [holiday]` though for example?

**Comment by t3chguy** [comment]
>  iirc metadata related events (rather than messages) like name changes don't inform the room list decoration, am I misremembering?

Yes, they do - anything unread will trigger a grey dot on a room list tile.

> My assumption was that whatever logic we're using there would be mirrored here.

Sure but we already use different logic for the app tile so this feels a bit strange
I think https://github.com/vector-im/element-web/issues/16836 tracks it sufficiently, will add a border to the dots to visually repair them.

**Comment by nadonomy** [comment]
> Yes, they do - anything unread will trigger a grey dot on a room list tile.

Gotcha. I thought we whitelisted specific events (messages/media uploads/calls/etc) to inform decoration but I must've been misremembering.

> will add a border to the dots to visually repair them.

For this issue/the beta staying the course on visual tweaks and not opening pandoras box on decoration logic sounds good to me!

**Comment by t3chguy** [comment]
> Gotcha. I thought we whitelisted specific events (messages/media uploads/calls/etc) to inform decoration but I must've been misremembering.

That was the plan, but it was part of FTUE Notifications which stalled out.
